server/lambda/layer/common/nodejs/custom_modules/send-to-sbc.js

```diff
@@ -1,28 +1,25 @@
-const dayjs = require("dayjs");
-const asyncPool = require("tiny-async-pool");
-const { postServiceItem } = require("./service-bc-api");
-
-const  FAIL = 'fail',
-const SUCCESS = 'success'
+const dayjs = require('dayjs');
+const asyncPool = require('tiny-async-pool');
+const { postServiceItem } = require('./service-bc-api');
 
 const getUnsuccessfulSbcTransactions = async (collection, arrivalKey) => {
   const dateRange = [
-    dayjs().subtract(13, "day"),
-    dayjs().subtract(3, "day"),
-  ].map((d) => d.startOf("day").toDate());
+    dayjs().subtract(13, 'day'),
```

Comment:
The last merge https://github.com/bcgov/enhanced-travel-screening/pull/158 - did not catch these somehow, running lint fixes some formatting changes